src/Data/Common/ImageHelper.ts

```diff
@@ -0,0 +1,62 @@
+import { Maybe } from "@/Core";
+
+interface FileSize {
+  size: number;
+}
+
+interface FileType {
+  type: string;
+}
+
+export class ImageHelper {
```

Comment:
With every method of this class being static, I wonder if it's useful having it as a class

Reply:
It is just an easy way to bundle these related methods into 1 container.